pandas/core/indexes/numeric.py

```diff
@@ -168,7 +168,7 @@ def _union(self, other, sort):
 ] = """
     Immutable ndarray implementing an ordered, sliceable set. The basic object
     storing axis labels for all pandas objects. %(klass)s is a special case
-    of `Index` with purely %(ltype)s labels. %(extra)s
+    of `Index` with purely %(ltype)s labels.
```

Comment:
Can you revert this one for now. It's a bit more complex, we'll take care of it in a separate PR.

Reply:
Done.